### ISSUE TITLE
graalVersion '1.0.0-rc6' wrong extraction directory

### ISSUE THREAD

**Issue by Xaseron** [palantir/gradle-graal#5]
Hey,

when i set the version of the graalvm to 1.0.0-rc6 the gradle plugin extracts it to the 1.0.0-rc5 directory.
But the plugin assumes the native-image binary of course in the 1.0.0-rc6 directory.
```
xaseron@arch ..le/caches/com.palantir.graal/1.0.0-rc6 % ls
graalvm-ce-1.0.0-rc5  graalvm-ce-1.0.0-rc6-amd64.tar.gz
```

My quick and dirty workaround is to manual extract the graalvm.

**Comment by markelliot** [comment]
Thanks for letting us know -- will take a look/am sure this is a relatively minor bug.

**Comment by markelliot** [comment]
Fixed in 0.1.1, thanks for reporting.
